python/test/t_NLopt_std.py

```diff
@@ -49,6 +54,6 @@ def printNumericalPoint(point, digits):
                     print('algo=', algo)
                     algo.run()
                     result = algo.getResult()
-                    print('x^=', printNumericalPoint(result.getOptimalPoint(), 4))
+                    #print('x^=', printNumericalPoint(result.getOptimalPoint(), 3))
```

Comment:
By the way, the current situation is worse: nothing is tested at all using the generic constuctor!